lib/request.coffee

```diff
@@ -190,6 +195,8 @@ module.exports = getRequest = ({ dataDirectory = null, debug = false, isBrowser
 	# 	stream.pipe(fs.createWriteStream('/opt/download'))
 	###
 	exports.stream = onlyIf(not isBrowser) (options = {}) ->
+		{ rindle } = loadDeps()
```

Comment:
This can just be `rindle = require('rindle')` - `require` already has the "once" semantics